install.sh

```diff
@@ -194,9 +190,39 @@ __install_bro() {
 	__install_packages bro broctl
 	chmod 2755 /opt/bro/logs
 	_BRO_PKG_INSTALLED=true
+	_BRO_INSTALLED=true
 	_BRO_PATH="/opt/bro/bin"
 }
 
+__configure_bro() {
+	# Configure Bro
+	wget -q "https://raw.githubusercontent.com/activecm/bro-install/master/gen-node-cfg.sh" -O "./gen-node-cfg"
```

Comment:
We should probably clean up these helper files after we're done with them or execute them from a tmpfs directory.